bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -179,6 +179,13 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
 
   CONFIG_PARAM(preExecutionResultAuthEnabled, bool, false, "if PreExecution result authentication is enabled");
 
+  CONFIG_PARAM(prePrepareFinalizeAsyncEnabled, bool, true, "Enabling asynchronous preprepare finishing");
+
+  CONFIG_PARAM(threadbagConcurrency,
+               uint32_t,
+               64u,
+               "Number of threads given to thread pool that is created for any request processing");
+
```

Comment:
It's not enough to just declare the parameter here. Please follow other parameters' declarations and do the same. Besides that, please don't forget to make it configurable through concord configuration_manager.

Reply:
yes I will be doing the same.

Reply:
I will also add the new configs in concord configuration_manager at the time of concord-bft bump.